x-pack/plugins/siem/public/components/page/hosts/events_table/index.test.tsx

```diff
@@ -58,13 +58,13 @@ describe('Load More Events Table Component', () => {
     test('formatIpSafely happy path', () => {
       const wrapperSourceIp = mountWithIntl(
         <ThemeProvider theme={theme}>
-          <p>{formatIpSafely('source.ip', mockData.Events.edges[0].node)}</p>
+          <p>{formatIpSafely('source.ip[0]', mockData.Events.edges[0].node)}</p>
```

Comment:
I am just wondering if later on, it will be useful that `formatIpSafely` should be able to deal with array directly. is it worth it to create an issue for that.

Reply:
Maybe? I haven't looked at the code closely.